### ISSUE TITLE
Unicode trouble with lemma_ still not fixed

### ISSUE THREAD

**Issue by NSchrading** [honnibal/spaCy#51]
As discussed in this issue: https://github.com/honnibal/spaCy/issues/32, printing lemmas of unicode words causes spaCy to crash. It was closed because I believe rsomeon did not want to contribute their patch, and instead preferred if you made your own changes. I don't believe the issue has been fixed, as I just tested it in v0.81 and I still get a crash.

```python
from spacy.en import English
s = "Fiancé"
nlp = English()
tok = nlp(s)
print(tok[0].lemma_)
```

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "spacy/tokens.pyx", line 585, in spacy.tokens.Token.lemma_.__get__ (spacy/tokens.cpp:10941)
  File "spacy/strings.pyx", line 73, in spacy.strings.StringStore.__getitem__ (spacy/strings.cpp:1671)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc3 in position 5: unexpected end of data
```

**Comment by honnibal** [comment]
Thanks, pushed a fix in 0.82

**Comment by NSchrading** [comment]
Yep, looks like it's working now. Thanks!
